webapp-django/crashstats/home/jinja2/home/home.html

```diff
@@ -21,46 +14,13 @@
 <div id="mainbody">
     <div class="page-heading">
         <h2 id="homepage-heading">{{ product }} Crash Data</h2>
-
-        <nav>
-            <ul id="duration" class="options">
-            {% for day in [3, 7, 14] %}
-                <li>
-                    <a  href="{{ change_query_string(days=day) }}"
-                        data-duration="{{ day }}"
-                        class="days-{{ day }}{% if day == days %} selected{% endif %}"
-                    >{{ day }} days</a>
-                </li>
-            {% endfor %}
-            </ul>
-        </nav>
     </div>
 
-    <div id="homepage-graph" class="panel">
-        <div class="title">
-            <h2>Crashes per 100 Active Daily Installs</h2>
-        </div>
-
-        <div
-            class="body"
-            id="homepage-graph-container"
-            data-supersearch-api="{{ url('api:model_wrapper', model_name='SuperSearch') }}"
-            data-product-build-types-api="{{ url('api:model_wrapper', model_name='ProductBuildTypes') }}"
-            data-adi-api="{{ url('api:model_wrapper', model_name='ADI') }}"
-            data-product="{{ product }}"
-            data-versions="{{ versions | to_json }}"
-            data-platforms="{{ platforms | to_json }}"
-            data-duration="{{ days }}"
-            data-es-shards-per-index="{{ es_shards_per_index }}"
-        >
-            <div class="message error hide"></div>
-            <div id="homepage-graph-graph"></div>
-            <div id="homepage-graph-legend" class="homepage-graph-legend"></div>
-        </div>
-    </div>
-
-
-    <div class="panel">
+    <div class="panel"
+        id="homepage-data-panel"
+        data-versions="{{ versions | to_json }}"
+        data-product="{{ product }}"
```

Comment:
These are both used by the remainder of the javascript, so I relocated them to this div.